holmes/admin/admin_api.py

```
@@ -0,0 +1,103 @@
+    try:
+        executor = _reload_and_rebuild_toolsets()
+        counts = _build_toolset_counts(executor)
+        return ReloadResponse(
+            status="ok",
+            component="toolsets",
+            detail=f"{counts['toolsets_total']} toolsets loaded, {counts['toolsets_enabled']} enabled, {counts['runbooks']} runbooks",
+            counts=counts,
+        )
+    except Exception as e:
+        logging.error("Failed to reload toolsets", exc_info=True)
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.15.11)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)